canonical/ubuntu.com
# WD-14397: add copy-to-clipboard to /pro/dashboard token

## Done

- Added functionality to copy token on click.

## QA

- Go to https://ubuntu-com-14254.demos.haus/pro/dashboard
- Click on the token in any subscription.
- Check if the token has been copied by pasting it somewhere.
- Check if "Token copied" notification has been displayed.

## Issue / Card

Fixes [WD-14397](https://warthogs.atlassian.net/browse/WD-14397) and #14210

## Screenshots

![image](https://github.com/user-attachments/assets/0eee31f6-7cf7-406f-a321-b11f91f9933c)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-14397]: https://warthogs.atlassian.net/browse/WD-14397?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
